tasks/bootstrap_local_vm/01_prepare_routing_rules.yml

```diff
@@ -24,6 +13,17 @@
   # vdsm installation breaks the routing by defining separate
   # routing table for ovirtmgmt. But we need to enable communication
   # between virbr0 and ovirtmgmt
+  - name: Start libvirt
```

Comment:
'include_tasks: alter_libvirt_default_net_configuration.yml' now is before this block but in alter_libvirt_default_net_configuration we are using virt_net module so I fear that libvirt should be already up and running before that.
I think that this is not a good approach.